scripts/run-session.sh

```diff
@@ -8,4 +8,8 @@ chmod 01777 /tmp/.X11-unix
 mkdir -p --mode=700 $XDG_RUNTIME_DIR
 
 export GNOME_SHELL_SESSION_MODE=ubuntu
+export PULSE_SERVER=unix:/run/user/`id -u`/pulse/native
+/usr/bin/pipewire &
```

Comment:
Why is it being run here and not as a service in gnome-session? A comment above this would be helpful.

Reply:
I think we want to run these as user session daemons as discussed here: https://github.com/canonical/core-base-desktop/pull/7#issuecomment-1411762668